flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -132,6 +132,7 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
       args.getIfNotNull<Boolean>("anrEnabled") { options.isAnrEnabled = it }
       args.getIfNotNull<Boolean>("sendDefaultPii") { options.isSendDefaultPii = it }
       args.getIfNotNull<Boolean>("enableNdkScopeSync") { options.isEnableScopeSync = it }
+      args.getIfNotNull<Boolean>("enableProfiling") { options.isProfilingEnabled = it }
```

Comment:
That alone isn't enough, since we don't enable performance by default on the Native layer.
My suggestion is: if `enableProfiling` is given, set the `tracesSampleRate` to the options as well.

Ps: This will only work if people `startTransaction` on the Native layer or using auto instrumentation for Native screens, transactions started on the Dart layer won't contain any profiling data, this is something we can improve later.